packages/catalog-model/examples/components/invalid-component.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: invalid component
```

Comment:
```suggestion
  name: invalid component  # wrong name convention 
```

Reply:
I did this with the description of the entity rather than as a commen.